packages/console/src/components/DeleteConfirmModal/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ReactNode, useState } from 'react';
+
+import ConfirmModal from '@/components/ConfirmModal';
+import TextInput from '@/components/TextInput';
+
+type Props = {
+  isOpen: boolean;
+  isLoading?: boolean;
+  children: ReactNode;
+  expectedInput?: string;
+  inputPlaceholder?: string;
```

Comment:
nit: not sure, but do you think wrapping them up into a typed object could be more self-contained? 
e.g.
```
type Captcha = {
 captchaValue: string;
 placeholder: string;
}


{ captcha && <TextInput />}
```

Reply:
I'm good with both

Reply:
I thought the placeholder could be optional, so it's ok to keep what it is.